src/pages/dashboard/NewReportPage.jsx

```diff
@@ -0,0 +1,244 @@
+"use client"
+
+import { useState } from "react"
+import { useNavigate } from "react-router-dom"
+import { MapPin, Upload, X } from "lucide-react"
+import { toast } from "sonner" // Cambiado a importar directamente de sonner
+
+import { Button } from "@/components/ui/button"
+import { Input } from "@/components/ui/input"
+import { Label } from "@/components/ui/label"
+import { Textarea } from "@/components/ui/textarea"
+import { Card, CardContent, CardDescription, CardFooter, CardHeader, CardTitle } from "@/components/ui/card"
+import DashboardLayout from "@/components/layout/DashboardLayout"
+import { getCurrentUser, createContaminationReport } from "@/services/data-services"
+
+export default function NewReportPage() {
+  const user = getCurrentUser()
+  const navigate = useNavigate()
+
+  const [title, setTitle] = useState("")
+  const [description, setDescription] = useState("")
+  const [lat, setLat] = useState("")
+  const [lng, setLng] = useState("")
```

Comment:
podrias encapsular esta logica en un solo state
```suggestion
  const [data, setData] = useState({ title: "", description: "", lat: "", lng: "" })
```